test/Microsoft.DocAsCode.Common.Tests/RelativePathTest.cs

```diff
@@ -415,5 +415,22 @@ public void TestUrlDecode(string path, string expected)
         {
             Assert.Equal(expected, ((RelativePath)path).UrlDecode());
         }
+
+        [Theory]
+        [InlineData("a/b/c", "a/b/", true)]
+        [InlineData("a/b", "a/b", true)]
+        [InlineData("a/b/", "a/b", true)]
+        [InlineData("a/b/c", "a/b/c", true)]
+        [InlineData("a/b/c", "a/b/c/d", false)]
+        [InlineData("a/b/c", "a/b/d", false)]
+        [InlineData("../a/b", "../a", false)]
```

Comment:
>"../a/b", "../a" [](start = 20, length = 16)

what is the result for `a/../b/c` and `b`?

Reply:
It's true. `../` will pop out `a/` in parts stack when constructing RelativePath.